portfolio/pom.xml

```diff
@@ -23,8 +23,14 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+    <groupId>com.google.code.gson</groupId>
+  <artifactId>gson</artifactId>
+  <version>2.8.6</version>
+  </dependency>
```

Comment:
Fix indentation

Reply:
ok, I fixed in on week2_3_fetch branch, then I git rebase